crates/viewer/re_selection_panel/src/selection_panel.rs

```diff
@@ -635,8 +635,19 @@ fn entity_selection_ui(
         .cloned();
 
     if let Some(view) = viewport.view(view_id) {
-        let view_ctx = view.bundle_context_with_states(ctx, view_states);
-        visualizer_ui(&view_ctx, view, entity_path, ui);
+        let class = view.class(ctx.view_class_registry());
+        view_states.ensure_state_exists(*view_id, class);
+        let view_state = view_states
+            .get(*view_id)
+            .expect("State got created just now"); // Convince borrow checker we're not mutating `view_states` anymore.
+        let view_ctx = view.bundle_context_with_state(ctx, view_state);
+
+        let empty_errors = PerVisualizer::default();
+        let visualizer_errors = view_states
+            .visualizer_errors(*view_id)
+            .unwrap_or(&empty_errors);
```

Comment:
```suggestion
        let empty_errors = PerVisualizer::default();
        let visualizer_errors = view_states
            .visualizer_errors(*view_id)
            .unwrap_or_default();
```

Might be missing something, but would this work as well?

Reply:
no, we're getting a reference here because I didn't want to copy things out.
Guess a `clone` wouldn't be too bad though 🤷